iib/workers/tasks/build.py

```diff
@@ -1077,7 +1077,35 @@ def handle_rm_request(
                 operators=operators,
                 binary_image=prebuild_info['binary_image'],
                 overwrite_from_index_token=overwrite_from_index_token,
+                generate_cache=False,
+            )
+
+            # generated by opm_migrate call inside opm_registry_rm_fbc
+            # rename `catalog` dir (because we need to use this name for
+            # final destination of catalog
+            catalog_from_db = os.path.join(temp_dir, 'from_db')
+            os.rename(os.path.join(temp_dir, 'catalog'), catalog_from_db)
+
+            # get catalog with opted-in operators
+            catalog_from_index = get_catalog_dir(
```

Comment:
adding `os.makedirs(os.path.join(temp_dir, 'from_index'))` before this line does not give the error mentioned in PR